server/project/project.proto

```diff
@@ -39,9 +39,10 @@ message ProjectTokenCreateRequest {
 // ProjectTokenResponse wraps the created token or returns an empty string if deleted.
 message ProjectTokenResponse {
     string token = 1;
+    int64 expiresAt = 2;
+    string id = 3;
```

Comment:
@tetchel FYI, ID and expiresAt are already encoded in the token JWT. Do you think it's too inconvenient to decode it from the token or do you feel like it needs to be surfaced in the response?

Reply:
It does not look like the Expires At is encoded but I suppose I could just leave that out of this output.

![image](https://user-images.githubusercontent.com/8888438/96941409-577c0300-14a0-11eb-9074-2f4819d28ed5.png)

Reply:
would you like me to make that change?

Reply:
The expiry time is only encoded if explicitly set - i.e. a token created with `--expires-in` option.

Reply:
that make sense, a null expiry means it never expires.

OK, doing the decode on the client-side makes sense then.